Common/Data/Consolidators/CalendarType.cs

```diff
@@ -22,44 +22,11 @@ static public class CalendarType
         /// <summary>
```

Comment:
I think we should add an `[Obsolete("CalendarType is obsolete, please use Calendar)]` message (or similar) for this class